### PR TITLE
fix: Updated misc names and values of semantic tokens

### PR DESCRIPTION
**Pull request by magnuh**
Updated names and values of semantic tokens according to changes in this document (greens added or changed, reds deleted):
https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit?pli=1#gid=888578831

### CODE REVIEW

File: tokens/blocket.se/semantic.yml [hunk 1]
```diff
@@ -7,155 +7,136 @@ s:
...
     border:
       _: bluegray-300
-      hover: gray-500
-      active:
-        _: gray-700
-        hover: gray-800
+      hover: bluegray-400
+      active: bluegray-500
       disabled: bluegray-300
+      inverted: white
```
Comment (BalbinaK) [inline tokens/blocket.se/semantic.yml:79]: I couldn't find this one in the spreadsheet.

File: tokens/blocket.se/semantic.yml [hunk 2]
```diff
@@ -169,12 +150,11 @@ s:
...
-        active: blue-600
-        disabled: bluegray-300
+      link: blue-600
+      disabled: bluegray-300
       negative: red-600
       positive: green-600
 
     focused: aqua-400
+
+    divider: bluegray-200
```
Comment (BalbinaK) [inline tokens/blocket.se/semantic.yml:160]: I didn't find this in the spreadsheet either but I suppose it was needed 😄

**Review by BalbinaK — APPROVED** [review]
No wonder why you can't stop having token nightmares 🤪 I got dizzy after just reviewing this PR 😂 Great job! 🚀

File: tokens/blocket.se/semantic.yml (hunk 1 above)
Comment (magnuh) [reply tokens/blocket.se/semantic.yml:79]: It will be introduced there as well 👍 😜

File: tokens/blocket.se/semantic.yml (hunk 2 above)
Comment (magnuh) [reply tokens/blocket.se/semantic.yml:160]: It was from the other document. 😅  ...but the whole divider thing might change. But we'll keep this one for now. 🙂

**Pull request merged**